ui/src/app/components/support.tsx

```diff
@@ -10,8 +12,21 @@ export const SupportMailto = ({ label = SUPPORT_EMAIL, style = {} }) => (
   </StyledExternalLink>
 );
 
-export const SupportButton = ({ label = SUPPORT_EMAIL, style = {} }) => (
-  <Button style={style} onClick={() => window.open(`mailto:${SUPPORT_EMAIL}`)}>
+const handleClickSupportButton = (url) => () => {
+  if (url) {
```

Comment:
this logic is now repeated - would it make sense to reuse some code here?

Reply:
Yeah, I wonder if this functionality should live in the urls utils file.